test/functional/README.md

```diff
@@ -99,6 +99,16 @@ P2PInterface object and override the callback methods.
 Examples tests are [p2p_unrequested_blocks.py](p2p_unrequested_blocks.py),
 [p2p_compactblocks.py](p2p_compactblocks.py).
 
+#### Prototyping tests
+
+The [`TestShell`](test-shell.md) class exposes the BitcoinTestFramework
+functionality to interactive Python3 environments, and can be used to prototype
```

Comment:
nit: remove comma after environments, as the clause after is not independent.

Reply:
Fixed.